README.md

```diff
@@ -60,6 +60,7 @@ Options:
       --cache                Only generate .d.ts and .d.ts.map for changed files.                                  [boolean] [default: true]
       --cacheStrategy        Strategy for the cache to use for detecting changed files.[choices: "content", "metadata"] [default: "content"]
       --logLevel             What level of logs to report.                            [choices: "debug", "info", "silent"] [default: "info"]
+  -o, --outputFolder         Output folder for generated files.                                                                     [string]
```

Comment:
NITS: Please update the name and description of this option.

Reply:
https://github.com/mizdra/happy-css-modules/pull/270/commits/fc1fdd64b09b848037f80a1659dbd951d1519e1c